tests/acquisition/covidcast/test_covidcast_meta_cache_updater.py

```diff
@@ -2,73 +2,234 @@
+        Uses a piece of the real data from the API to compare the results.
+
+        test-data1.csv was generated with 
+
+        >>> start_day = date(2020, 1, 20)
+        >>> end_day = date(2020, 5, 1)
+        >>> df = pd.concat([ 
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_cumulative_num", start_day=start_day, end_day=end_day, time_type="day", geo_type="state", geo_values="ak,ca"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_cumulative_num", start_day=start_day, end_day=end_day, time_type="day", geo_type="county", geo_values="02100,06003,06017,06027"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_incidence_num", start_day=start_day, end_day=end_day, time_type="day", geo_type="state", geo_values="ak,ca"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_incidence_num", start_day=start_day, end_day=end_day, time_type="day", geo_type="county", geo_values="02100,06003,06017,06027"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_7dav_incidence_num", start_day=start_day, end_day=end_day, time_type="day", geo_type="state", geo_values="ak,ca"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_7dav_incidence_num", start_day=start_day, end_day=end_day, time_type="day", geo_type="county", geo_values="02100,06003,06017,06027")
+        ])
+        >>> df.to_csv("test-data.csv")
+
+        test-data2.csv was generated with (just a start_day difference, to match with 7dav)
+
+        >>> end_day = date(2020, 5, 1)
+        >>> df = pd.concat([ 
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_cumulative_num", start_day=date(2020, 2, 13), end_day=end_day, time_type="day", geo_type="state", geo_values="ak,ca"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_cumulative_num", start_day=date(2020, 2, 13), end_day=end_day, time_type="day", geo_type="county", geo_values="02100,06003,06017,06027"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_incidence_num", start_day=date(2020, 2, 14), end_day=end_day, time_type="day", geo_type="state", geo_values="ak,ca"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_incidence_num", start_day=date(2020, 2, 14), end_day=end_day, time_type="day", geo_type="county", geo_values="02100,06003,06017,06027"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_7dav_incidence_num", start_day=date(2020, 2, 20), end_day=end_day, time_type="day", geo_type="state", geo_values="ak,ca"),
+            covidcast.signal(data_source="jhu-csse", signal="confirmed_7dav_incidence_num", start_day=date(2020, 2, 20), end_day=end_day, time_type="day", geo_type="county", geo_values="02100,06003,06017,06027")
+        ])
+        >>> df.to_csv("test-data.csv")
+        """
+        self._insert_csv("repos/delphi/delphi-epidata/tests/acquisition/covidcast/test-data/test-data2.csv")
```

Comment:
There's a testdata folder in the delphi-epidata folder. It may allow for easier reuse of these flat files if needed for other tests.

Reply:
Another note on this. The directory structure here is assuming that the base working directory is above the repos folder. This is inconsistent with the t[est_utils.py](https://github.com/cmu-delphi/delphi-epidata/blob/dev/src/acquisition/covid_hosp/common/test_utils.py)

Reply:
Hm, I've never even looked at that file before. Looks like it's all for tests of `covid_hosp`, which I haven't really touched. 

Wait, it looks like `test_utils.py` does some complicated directory traversing up the tree until it can find `testdata`.

Reply:
Yeah you’re right. Let me dig through the source to see if that’s used elsewhere.